docs/settings.rst

```diff
@@ -0,0 +1,142 @@
+.. _settings:
+
+--------
+Settings
+--------
+
+Wafer has several Django settings that control its behaviour.
+It attempts to provide reasonable defaults for these, (and Django in general),
+in the ``wafer.settings`` module, so you can import this in your app's
+``settings.py``, and then override things you want to change.
+
+
+Wafer's settings
+================
+
+``PAGE_DIR``
+    The directory that the ``load_pages`` management command will load
+    pages from.
+    Should be an absolute path with a trailing ``/``.
+
+``TALKS_OPEN``
+    Controls whether talk submissions are accepted.
+    Set to ``False`` to close talk submissions.
+
```

Comment:
Do we want to mention that there's fine grained control avaible for individual talk types via the admin interface? Not sure if this is the best place to mention that, though

Reply:
I think we should have a page on conference paper management.